jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java

```diff
@@ -3,25 +3,179 @@
 import io.github.epam.TestsInit;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.buttonFrame;
-import static io.github.com.StaticSite.inputButtonDefaultPage;
-import static io.github.com.StaticSite.inputButtonDisabledPage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.inputs.ButtonPage.*;
 
 public class ButtonTests extends TestsInit {
 
     @Test
     public void defaultButtonTest() {
-        inputButtonDefaultPage.open();
+        openSection("Buttons");
 
-        buttonFrame.button.is().enabled();
-        buttonFrame.button.click();
-        buttonFrame.button.has().text("BUTTON");
+        button.getContainedButtonByIndex(1).click();
```

Comment:
Please use method like getButtonByGroupIndex(String group, int index), where group could be "Contained", "Text" and etc.

Reply:
done